src/model.d.ts

```diff
@@ -1703,6 +1703,11 @@ export interface ModelOptions<M extends Model = Model> {
    */
   omitNull?: boolean;
 
+  /**
+   * Adds primary key automatically to the model if false. Default false.
```

Comment:
use `@default` for default values, TypeDoc will render it :)

```suggestion
   * Sequelize will automatically add a primary key called `id` if no
   * primary key has been added manually.
   *
   * Set to false to disable adding that primary key.
   *
   * @default false
```